cookbook/case_studies/feature_engineering/feast_integration/feast_workflow.py

```diff
@@ -1,5 +1,5 @@
 """
-Flyte Pipeline with Feast
+Flyte Pipeline With Feast
```

Comment:
"with" should be lower case in this particular case https://capitalizemytitle.com/ufaqs/do-you-capitalize-with-in-a-title/#:~:text=According%20to%20the%20Chicago%20Manual,means%20it%20should%20be%20lowercase.

Reply:
In the APA style title case, "with" has to be "With", whereas in the Chicago style title case, "with" has to be lowercase.